src/main/java/com/google/devtools/build/lib/worker/WorkerModule.java

```diff
@@ -176,7 +184,7 @@ public void buildStarting(BuildStartingEvent event) {
     }
 
     // Override the flag value if we can't actually use cgroups so that we at least fallback to ps.
-    boolean useCgroupsOnLinux = options.useCgroupsOnLinux && CgroupsInfo.isSupported();
+    boolean useCgroupsOnLinux = options.useCgroupsOnLinux && rootCgroup.memory() != null;
```

Comment:
nit: Let's use some method like `hasMemoryController()` instead?

Reply:
This will need to change anyways if we want to start collecting cpu usage as well